script/bootstrap

```diff
@@ -15,10 +15,13 @@ echo
 SHELL_CONFIG="$HOME/.zshrc"
 touch $SHELL_CONFIG
 
-# Check if RSA SSH keys already exist
+# Check if RSA or Ed25519 SSH keys already exist
 if [[ -f ~/.ssh/id_rsa ]]; then
   echo "==> Found SSH key in ~/.ssh/id_rsa"
   echo
+elif [[ -f ~/.ssh/id_ed25519 ]]; then
```

Comment:
huh, TIL https://medium.com/risan/upgrade-your-ssh-key-to-ed25519-c6e8d60d3c54. Thanks!